json_schema/type/project/project.json

```
@@ -77,6 +77,28 @@
             "user_friendly": "INSDC project accession",
             "guidelines": "Enter accession if project has been archived. Accession can be from the DDBJ, NCBI, or EMBL-EBI and must start with DRP, SRP, or ERP, respectively."
         },
+        "ega_accessions": {
+            "description": "A list of European Genome phenome Archive dataset or study accessions.",
```

this is a list because 1 HCA project = many EGA studies?

---

sorry, you probably mentioned this but just verifying to be absolutely sure

---

Yep! there may be cases where 1 HCA project == Many EGA accessions

---

Capitalization and phrasing is odd.
```suggestion
            "description": "A list of accessions referring to EGA (European Genome-Phenome Archive) datasets or studies.",
```